LIPS/lip-13.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Easy Track payments limit 
+tags: LIPs
+blockchain: ethereum
+lip: lip-13
+status: Implemented
+author: Alexey Potapkin, Eugene Pshenichnyy, Victor Suzdalev, Eugene Mamin
+discussions-to: https://research.lido.fi/t/lip-13-easy-track-payments-limit/1670
+---
+
+# Easy Track payment limits
+
+## Simple summary
+
+A simple security rule that limits Easy Track Motions while moving treasury funds.
+
+## Abstract 
+
+We propose adding the Aragon ACL permission that will limit an asset type and its amount for each payment initiated by Easy Track. The permission constrained with the limits will be set by DAO vote.
```

Comment:
I suggest to rewrite this as follows:
"We propose to add the Aragon ACL permission that will whitelist only required asset types and limit their amounts for each payment initiated by Easy Track. The permission is set and updated only by explicit DAO vote."